pkg/ebpf/ingress_node_firewall_loader_test.go

```diff
@@ -12,11 +12,9 @@ func TestAddOrUpdateRules(t *testing.T) {
 	// Skip this test unless someone wants to run it explicitly. This test here can also probably be dropped completely
 	// after the below if index issue (see FIXME below) is fixed. The ebpfsyncer tests test the same code just from
 	// a higher level.
-	t.Skip()
```

Comment:
Keep this skip here. When I run `make test` as root, the cleanup logic in [ingress_node_firewall_loader_test.go] races sometimes with the tests run in ebpfsyncer_test.go  (the tests from both files seem to "overlap" sometimes). The tests in ebpfsyncer_test.go are more thorough and test way more, tbh I only added the test in ingress_node_firewall_loader_test.go because of this issue and because I wanted a low level test.

Later, if someone is interested in writing unit tests for the ingress_node_firewall_loader, they can figure out how to run those tests properly together.